scrunch/datasets.py

```diff
@@ -2412,14 +2409,22 @@ def fork(self, description=None, name=None, is_published=False,
             **kwargs
         )
 
+        owner = kwargs.get("owner")
         if preserve_owner:
             body['owner'] = self.resource.body.owner
```

Comment:
If they choose to preserve owner, what they mean is that it should be in the same place as the current project. The API already returns the project URL for the `owner` attribute. So this may also be `self.resource.project`.

And we can slowly stop the conversaion about owners, because that concept does not exist in the API anymore and will bring confusion to users that individuals have ownership of datasets.

Reply:
**question:** @jjdelc I am a bit confused here - We want to keep the `preserve_owner` kwarg and add a new one, `project`, explicitly to the signature by your suggestion; then basically fall back to `self.resource.project` (simple code change to abandon the old  `owner` property) if `project` is not given?

Reply:
Yes, really at this point owner and project are equivalent. There is no concept of an individual owner in Crunch.

It looks like `preserve_owner` is the default behavior when no owner is passed in, if preserve_owner is False, and no owner is passed, then there's no explicit code path for this.

Reply:
Thanks for confirming.